js/coinbase.js

```diff
@@ -104,64 +104,117 @@ module.exports = class coinbase extends Exchange {
             },
             'api': {
                 'public': {
```

Comment:
@Dan-krm why not group all methods from a version inside the same object? (I think that is the "standard" way of structuring our APIs)

```
v2: {
  public: {
  }
  private: {
   }
}
v3: {
  public: {
  }
  private: {
   }
}
```

Reply:
@Dan-krm thanks 🙂